.github/workflows/actions.yml

```diff
@@ -5,22 +5,80 @@ on:
   pull_request:
     branches: [master]
 jobs:
-  build:
-    name: ruby${{ matrix.ruby }} rails${{ matrix.rails }} rake
+  test_rails_7_2:
+    name: Ruby 3.3, Rails 7.2
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ruby: ['2.7', '3.0']
-        rails: ['5.2', '6.0', '6.1', '7.0', '7.1']
-        exclude:
-        - ruby: '3.0'
-          rails: '5.2'
```

Comment:
if you don't want to deal with bad combinations then do:
```
include:
  - rails: 2.7
     ruby: 3.0
```
or something like that, but I'd prefer that over copy-paste solutions

Reply:
can drop 2.7/3.0 and 5.2 + 6.0 if that makes things easier